source/heating_model/interface.cc

```diff
@@ -171,6 +171,11 @@ namespace aspect
         model_names
           = Utilities::split_string_list(prm.get("List of model names"));
 
+        AssertThrow(Utilities::list_is_unique(model_names),
+                    ExcMessage("The list of strings for the parameter "
+                               "'Heating model/List of model names' is not unique. "
```

Comment:
The list is unique alright -- it's just that some entries appear twice :-)

How about "The list of strings for the parameter ... contains entries more than once. This is not allowed."?